### PR TITLE
bpo-20039: Add doc to argparse.ArgumentTypeError

### PR DESCRIPTION
**Pull request by ukarroum**
https://bugs.python.org/issue20039

### CODE REVIEW

**Review by MaxwellDupre — APPROVED** [review]
I think this adds value.

**Review by nanjekyejoannah — COMMENTED** [review]
If so do we need to document `ArgumentError` as well too?

File: Doc/library/argparse.rst [hunk 1]
```diff
@@ -2130,6 +2130,13 @@ containing the populated namespace and the list of remaining argument strings.
...
 remaining unparsed argument strings.
 
+Exceptions
+^^^^^^^^^^
+
+.. exception:: ArgumentTypeError
+
+    An error from trying to convert a command line string to a type
+
 .. versionadded:: 3.7
```
Comment (slateny) [inline Doc/library/argparse.rst]: This line should be above the 'Exceptions' header.

Comment (ukarroum) [reply Doc/library/argparse.rst]: Fixed

**Comment by ukarroum** [comment]
Looks like this issue was already solved in this commit: 79ae019164eeb6b94118bc17bc1e937405684c75
closing the pull request

**Pull request closed without merging**